### DISCUSSION TITLE
Incremental storage backup using velero on EKS PV

### DISCUSSION THREAD

**Discussion by vaibyada** [vmware-tanzu/velero#7580] [General discussions]
I am using velero for backup and restore for EKS solution and has PV's associated with it. Does velero supports incremental backup as it takes snapshot of volumes attached to node and snapshot is always full backup.

If it supports incremental backup then how to configure full backup every Sunday and incremental on rest of the day and this cycle should continue. So every Sunday Full and Monday to Saturday incremental and so on...

**Comment by blackpiglet** [comment]
AFAIK, Velero doesn't support the incremental snapshot backup.
This depends on the upstream CSI CBT support.

**Reply by vaibyada** [reply]
So there is no way in my case that I will be able to take incremental backup of my EKS persistent volumes? If yes then can you please guide for any documentation.

**Reply by blackpiglet** [reply]
The filesystem backup supports the incremental way, but the filesystem backup can support the incremental way. https://velero.io/docs/v1.13/file-system-backup/
Please check whether that is what you need.
